src/style/measureText.js

```diff
@@ -0,0 +1,23 @@
+import rnTextSize from 'react-native-text-size';
```

Comment:
What about adding this functionality to `Typography` instead of having a new class. 
It'll be similar to `Colors` that has our presets and util functions altogether.

Reply:
But if you don't have the native linked won't you crash when importing typography in your screen?

Reply:
we can require it inside the method anyway, using it will crash, but just importing it won't (I think) 
which I'm fine, we can wrap with try catch and throw nice error that says you need to link the module or something..

Reply:
require inside the method? you mean `const textMeasure = require('react-native-text-measure';`?